detections/endpoint/windows_svchost_exe_parent_process_anomaly.yml

```diff
@@ -0,0 +1,76 @@
+name: Windows Svchost.exe Parent Process Anomaly
+id: 1d38e5e9-2ff8-4c47-872c-bf1657cefab5
+version: 1
+date: '2025-02-11'
+author: Teoderick Contreras, Splunk
+status: production
+type: Anomaly
+description: The following analytic detects an anomaly where a svchost.exe process is spawned by a parent process other than the standard services.exe. In a typical Windows environment, svchost.exe is a system process that hosts Windows services and is expected to be a child of services.exe. A process deviation from this hierarchy may indicate suspicious behavior, such as malicious code attempting to masquerade as a legitimate system process or evade detection. It is essential to investigate the parent process and associated behavior for further signs of compromise or unauthorized activity.
+data_source:
+- Sysmon EventID 1
+- Windows Event Log Security 4688
+search: '| tstats `security_content_summariesonly` count min(_time) as firstTime max(_time) as lastTime from datamodel=Endpoint.Processes 
+  where Processes.parent_process_name != "services.exe" AND Processes.process_name = "svchost.exe" AND Processes.process != unknown
+  by Processes.parent_process_name Processes.parent_process_path Processes.parent_process Processes.process_path Processes.process Processes.original_file_name Processes.dest Processes.user 
```

Comment:
`MsMpEng` from Windows Defender is a known FP to this case, so please add it.

Also please specify the full path of `services.exe` instead of just the name to avoid false negatives.

MsMpEng exists in a couple of paths, for example

- `C:\ProgramData\Microsoft\Windows Defender\Platform\*\MsMpEng.exe`
- `C:\Program Files\Windows Defender\MsMpEng.exe`

Reply:
noted on that